samples/nrf9160/pdn/README.rst

```diff
@@ -59,14 +59,33 @@ The sample shows the following output, which may vary based on the network provi
 
    PDN sample started
    Event: PDP context 0 activated
-   Created new PDP context 1
+   Event: PDP context 0 IPv6 up
    Default APN is telenor.iot
+   Created new PDP context 1
    PDP context 1 configured: APN telenor.iot, Family IPV4V6
-   Event: PDP context 0 IPv6 up
    Event: PDP context 1 activated
    Event: PDP context 1 IPv6 up
-   PDP Context 0 (cid)      PDN ID 0
-   PDP Context 1 (cid)      PDN ID 0
+   PDP Context 0, PDN ID 0
+   PDP Context 1, PDN ID 0
+
+   Interface addresses:
+   Address: 10.118.20.13
+   Netmask: 0.0.0.0
+   Broadcast address: 0.0.0.0
+   Destination address: 0.0.0.0
+
+   Address: 0:0:fe80::11
```

Comment:
This doesn't look right, and the netmask, etc don't look correct either, for IPv6

Reply:
Address issue is due to typo in case checking of family (see below). Netmask, etc, are correctly displayed as reported (they are just not set by the modem).